api/src/main/java/com/google/sps/SendNotifications.java

```diff
@@ -95,46 +95,45 @@ public Email getNextRecipient(HttpServletRequest request, HttpServletResponse re
   public void sendEmail(HttpServletRequest request, HttpServletResponse response)
       throws IOException, InterruptedException, ExecutionException {
 
-    Firebase.init();
-    Firestore db = FirestoreClient.getFirestore();
-
     EmailType emailType = EmailType.valueOf(request.getParameter("emailType").toUpperCase());
-    String gameID = request.getParameter("gameID");
     final String FROM = "pictophone.noreply@gmail.com";
 
-    DocumentSnapshot docSnap = db.collection("games").document(gameID).get().get();
+    switch(emailType) {
+      case END:
+        List<Email> emails = gatherRecipients(request, response);
 
-    if (emailType == EmailType.START || emailType == EmailType.END) {
-      List<Email> emails = gatherRecipients(request, response);
+        try {
+          Gmail service = ServiceCreation.createService();
 
-      try {
-        Gmail service = ServiceCreation.createService();
+          for (Email email : emails) {
+            MimeMessage encoded = createEmail(email.getEmail(), FROM, email.getSubject(), email.getBody());
+            Message testMessage = sendMessage(service, FROM, encoded);
 
-        for (Email email : emails) {
-          MimeMessage encoded = createEmail(email.getEmail(), FROM, email.getSubject(), email.getBody());
-          Message testMessage = sendMessage(service, FROM, encoded);
-
-          System.out.println("Email: " + testMessage.toPrettyString());
+            System.out.println("Email: " + testMessage.toPrettyString());
+          }
+        } catch (Exception e) {
+          System.out.println("Sending Email Exception: " + e);
+          System.err.println(e);
         }
-      } catch (Exception e) {
-        System.out.println("Sending Email Exception: " + e);
-        System.err.println(e);
-      }
-    } else if (emailType == EmailType.TURN) {
-      Email player = getNextRecipient(request, response);
+        break;
+      case TURN:
+        Email player = getNextRecipient(request, response);
 
-      try {
-        Gmail service = ServiceCreation.createService();
+        try {
+          Gmail service = ServiceCreation.createService();
```

Comment:
Is a new Gmail service required for every new email? Is it possible to create a static global Gmail service upon server start, just like Firebase?